mintlify/writer
# Mono Repo Time

- `vscode/` contains the VS Code extensions
- `intellij/` contains the IntelliJ plugin
- `server/` contains the server setup (Connect stuff removed)

### Testing instructsion
- Ensure that licenses are in place and correctly stated
- Ensure that the backend runs properly
- VSCode extension works when you open VS Code in `vscode/` as the root
- IntelliJ plugin works when you open IntelliJ IDEA in in `intellij/`
